src/internal_modules/roc_pipeline/receiver_session.cpp

```diff
@@ -246,6 +246,7 @@ audio::IFrameReader& ReceiverSession::frame_reader() {
 status::StatusCode ReceiverSession::route_packet(const packet::PacketPtr& packet) {
     roc_panic_if(!is_valid());
 
+    packet->udp()->enqueue_ts = core::timestamp(core::ClockUnix);
     return packet_router_->write(packet);
```

Comment:
1. Maybe move this into packet::Router::write? Packet router is supposed to be entry point to the packets pipeline, so it looks like a good place to be responsible for filling timestamps. On the other hand, it would be nice to keep pipeline more high-level.

2. We need a guard for the case when packet->udp() returns NULL. (I think we can just skip timestamp filling in this case).